grammar_checker.py

```diff
@@ -81,6 +81,8 @@ def strike(self,text):
     # collects grammar corrections and repeated words
     def tool(self,sentence):
         split = sentence.split()
+        if not split[-1][-1].isalpha(): #considering the punctuation at the end of the sentence
```

Comment:
Only considering the punctuation at the end of the sentence, should we also take into account possible punctuations inside the sentence?